spec/integration/spec_file_load_errors_spec.rb

```diff
@@ -99,6 +99,9 @@
           exit
         # ./helper_with_exit.rb:1#{spec_line_suffix}
       EOS
+    elsif RSpec::Support::Ruby.truffleruby?
+      expect(output).to include
+        "While loading ./helper_with_exit.rb an `exit` / `raise SystemExit` occurred, RSpec will now quit."
```

Comment:
The different output here seems unexpected, I think we should match CRuby behavior here.
Maybe the problem is `$!` isn't set in some case? (a random guess)

Reply:
failed an issue https://github.com/oracle/truffleruby/issues/2632

Reply:
This has been fixed, so this change shouldn't be needed anymore, could you remove it from this PR?

Reply:
This has been fixed, so this change shouldn't be needed anymore, could you remove it from this PR?

Reply:
yes, I will update the PR in next few days